doc/howto/format2/python_nose_configuration.rst

```diff
@@ -22,11 +22,10 @@ CMakeLists.txt
 Declare each nose test like this::
 
   if (CATKIN_ENABLE_TESTING)
-    catkin_add_nosetests(test_your_node tests/test_your_node.py)
+    catkin_add_nosetests(test_your_node.py)
```

Comment:
Actually the function should commonly be called with the basepath where all the nosetests are in, e.g. `tests`.